confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/ConfigPhysicalPlanType.java

```diff
@@ -65,6 +65,7 @@ public enum ConfigPhysicalPlanType {
   CreateFunction,
   DropFunction,
   GetNodePathsPartition,
+  CreateSchemaTemplate,
```

Comment:
It's all good to add new entry at middle, while it's better to append it at the end to keep enumerations compatible.

Reply:
append it at the end